frontend/packages/dev-console/src/components/monitoring/alerts/SilenceAlert.tsx

```diff
@@ -14,23 +20,51 @@ const { alertManagerBaseURL } = window.SERVER_FLAGS;
 
 const SilenceUntil = ({ rule }) => {
   if (!_.isEmpty(rule.silencedBy)) {
-    return <StateTimestamp text="Until" timestamp={_.max(_.map(rule.silencedBy, 'endsAt'))} />;
+    return (
+      <div onClick={(e) => e.preventDefault()} role="presentation">
+        <StateTimestamp text="Until" timestamp={_.max(_.map(rule.silencedBy, 'endsAt'))} />
+      </div>
+    );
   }
   return null;
 };
 
 const SilenceAlert: React.FC<SilenceAlertProps> = ({ rule }) => {
-  const [isChecked, setIsChecked] = React.useState(rule.state !== RuleStates.Silenced);
-  const [ruleState] = React.useState(rule);
-  React.useEffect(() => setIsChecked(ruleState.state !== RuleStates.Silenced), [ruleState]);
+  const [isChecked, setIsChecked] = React.useState(true);
+  const [isInprogress, setIsInprogress] = React.useState(false);
+  const rules = useSelector((state: RootState) => state.UI.getIn(['monitoring', 'devRules']));
+  const dispatch = useDispatch();
+
+  React.useEffect(() => {
+    if (rule.state === RuleStates.Silenced) {
+      setIsChecked(false);
+    }
+  }, [rule]);
 
   const handleChange = (checked: boolean) => {
+    setIsChecked(checked);
     if (checked) {
       _.each(rule.silencedBy, (silence) => {
-        coFetchJSON.delete(`${alertManagerBaseURL}/api/v2/silence/${silence.id}`);
+        coFetchJSON
+          .delete(`${alertManagerBaseURL}/api/v2/silence/${silence.id}`)
```

Comment:
I think this should be using `ALERT_MANAGER_TENANCY_BASE_PATH` instead of `alertManagerBaseURL`